docs/canvas/canvas-tutorial.asciidoc

```diff
@@ -79,23 +78,24 @@ You're now looking at the raw data syntax that Canvas uses to display the elemen
 
 .. Look for `math "sum_total_price"`, then add `| formatNumber "$0a"`.
```

Comment:
We need to rewrite this section of the docs. These steps don't actually work since I added the `metricFormat` argument to the `metric` functions in 7.4 https://github.com/elastic/kibana/pull/42007/files#diff-9ce33bafba483f99c24c92ad7c348ff5.

Maybe to something like this:
```suggestion
.. Look for `metricFormat="0,0.[000]"`, then change that to `metricFormat="$0a"`.
```